core/modules/messages/client/messages-popup.hbs.html

```diff
@@ -0,0 +1,21 @@
+<template name="messagesPopup">
+  <ul class="notifications">
+    {{#each message in messages}}
+      <div id={{message.id}} class="toast">
+        {{> avatarViewer user=message.user}}
+        <div class="right-content">
+          {{#if eq message.type 'global'}}
+            <p class="global-title">Global</p>
```

Comment:
"Global" looks weird and inadequate to me, especially if you send / receive a message in a particular zone. Let's put the zone name instead?